src/Utilities/hooks/useChromePush.js

```diff
@@ -5,7 +5,11 @@ export const useChromePush = () => {
     return (e, link) => {
         if (typeof push === 'function') {
             e.preventDefault();
-            push(link.replace(/^.\//, '/'));
+            if (insights.chrome.isBeta()) {
```

Comment:
I think we can remove the returned function as it is not needed. 

What this does now is whenever the dashboard unmounts, it either adds or removes `/beta`. I think this is not needed if we want to keep the environment the same across the navigation.